babel.config.js

```diff
@@ -47,12 +47,6 @@ module.exports = {
         development: {
             plugins: ['flow-react-proptypes'],
         },
-        npm: {
-            plugins: [['react-remove-properties', { properties: ['data-testid'] }]],
-        },
-        production: {
-            plugins: [['react-remove-properties', { properties: ['data-resin-target', 'data-testid'] }]],
```

Comment:
Should `data-resin-target` attributes still be removed? That change seems unrelated to `data-testid`. If not, the `react-remove-properties` dependency should also be removed.